admin/templates/base.html

```diff
@@ -0,0 +1,110 @@
+{% load staticfiles %}
+<script src="{% static 'vendor/bower_components/raven-js/dist/raven.min.js' %}"></script>
```

Comment:
I commented on this in @leb2dg's PR so it's probably just a merge thing but JS should be inside the html tag. These in particular should be right before the closing head tag.